_overviews/scala3-migration/incompat-type-checker.md

```diff
@@ -18,17 +18,19 @@ It is unsound and might cause runtime failures, as demonstrated by this [test](h
 
 The Scala 3 compiler does not permit this anymore.
 
-```scala
+{% tabs scala-3-unsound_vc_1 %}
+{% tab 'Scala 3 Only' %}
```

Comment:
There is confusion going on here, this code is not valid in Scala 3, so we should put “Scala 2 Only” in the tab title.